mysql-test/suite/innodb/t/temp_table_encrypt.test

```diff
@@ -0,0 +1,85 @@
+--source include/have_innodb.inc
+--source include/count_sessions.inc
```

Comment:
Don't count sessions here, you are restarting the server later - move it after the last in-testcase restart

Reply:
Right.. added some testcase scenarios at top and this is now placed at wrong position

Reply:
clarified point 3